Added Stargate Protocol

cryptonoyaiba opened the pull request:
- Added Stargate as CrossChainServiceProvider.
- Old branch - merged master and fixed conflicts.

Commit:
Added Stargate Protocol

- Added new example for testing Stargate Protocol integration
- Changed package.json version

Commit:
Update package-lock.json

- Changed version

0xSulpiride reviewed (approved):
LGTM

vignesha22 commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "etherspot",
-  "version": "1.35.2",
+  "version": "1.37.0",
```
shouldn't it be 1.36 as the next version available?

vignesha22 replied:
Okay got it so pls merge it after hop integration

lbw33 replied:
If Hop is merged first then (which is 1.36.0) then this should follow on from it

cryptonoyaiba committed:
Merge remote-tracking branch 'origin/master' into feature/PRO-305-integration-of-stargate-for-bridging

cryptonoyaiba committed:
Removed old example

Removed old Stargate example. Can now used modified version of example 16. Added run command for example 19.

lbw33 commented:
TODO: bump package version

cryptonoyaiba committed:
1.44.0
